### PR TITLE
Added 3 Lua player API properties

### PR DESCRIPTION
**Pull request by cjshmyr**
Added 3 Lua player API properties:
- `InternalName`
- `IsNonCombantant` for use with GetPlayers() to get only combatant players back (no Creeps/Neutral)
- `IsLocalPlayer` for individualized player code (such as setting camera position, or future Lua UI stuff)

Test sole survivor map/branch: https://github.com/cjshmyr/OpenRA/commit/e76cc7729e9895359149357f652c847cadbfa2fb

### CODE REVIEW

**Comment by pchote** [comment]
As discussed in IRC these are simple changes and we want SS to be playable in the next release &rarr; adding to milestone.

File: OpenRA.Mods.Common/Scripting/Properties/PlayerProperties.cs [hunk 1]
```diff
@@ -52,6 +55,12 @@ public string Race
 		[Desc("Returns true if the player is a bot.")]
 		public bool IsBot { get { return Player.IsBot; } }
 
+		[Desc("Returns true if the player is non combatant.")]
+		public bool IsNonCombatant { get { return Player.NonCombatant; } }
+
+		[Desc("Returns true if the player is the local player.")]
+		public bool IsLocalPlayer { get { return Player.PlayerActor.Owner == (Player.World.RenderPlayer ?? Player.World.LocalPlayer); }	}
```
Comment (obrakmann) [inline OpenRA.Mods.Common/Scripting/Properties/PlayerProperties.cs]: eh... I'm pretty sure that `Player` and `Player.PlayerActor.Owner` are the same.

**Comment by cjshmyr** [comment]
Added @obrakmann's correction

**Comment by obrakmann** [comment]
:+1:

**Comment by Mailaender** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/f5f4dbc26a9893187e3f2f10003fa9ab96c3d568...6fe26e772ef5a0ac5f7836b124fcf8819adf79eb)

**Pull request merged**